exercises/roman-numerals/description.md

```diff
@@ -19,22 +19,30 @@ The Romans wrote numbers using letters - I, V, X, L, C, D, M.
 ```
 
 The maximum number supported by this notation is 3,999.
-(The Romans themselves didn't tend to go any higher)
+(The Romans themselves didn't tend to go any higher.)
```

Comment:
I'd remove this line. Romans did and could go higher using alternatives forms of notation. https://mathworld.wolfram.com/RomanNumerals.html